apps/server/src/socket.ts

```diff
@@ -41,7 +41,60 @@ const onOfferRide =
       member: socket.data.user.uid,
     });
 
+    socket.on('driverLocation', async (location) => {
+      await redis.GEOADD('drivers', {
+        longitude: location.lng,
+        latitude: location.lat,
```

Comment:
I think files are getting too big,we can split them into two different files

Reply:
good point. I'll create a file for driver socket events and a file for rider socket events